src/controllers/user.js

```
@@ -14,7 +16,12 @@ module.exports = {
   deleteUser: (req, res) => {
     res.send('User with the specified ID has been deleted');
   },
-  getAllDonators: (req, res) => {
-    res.send('All donators');
+  getAllDonators: async (req, res) => {
+    try {
+      const donators = await UserModel.find({ isDonator: true });
+      res.json(donators);
```

The same issue as in items, you can throw an error here too.